routes/productRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const productController = require("../controllers/productController");
+
+const router = express.Router();
+
+router.get("/productInfo/:productId", productController.getProductDetail);
```

Comment:
최종 endpoint가 /products/:productId 만으로 충분 할 것 같습니다.
이렇게 하면 products(여러 product들) 중에 해당 Id를 가지고 있는 product를 의미하게 됩니다.

```suggestion
router.get("/:productId", productController.getProductDetail);
```